content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -72,8 +72,61 @@ In case your cluster nodes are customized and install additional security and
 telemetry agents on the node, make sure to check with the vendor of the agent whether it has dependency on Docker.
 
 ### Telemetry and security agent vendors
+This section is intended to collect information about various telemetry and security agents that may have dependency on containers runtime. Support matrix section outlines the supported runtimes and Migration from dockershim section is designed to help users transition from dockershim to other container runtimes.
 
 We keep the work in progress version of migration instructions for various telemetry and security agent vendors
 in [Google doc](https://docs.google.com/document/d/1ZFi4uKit63ga5sxEiZblfb-c23lFhvy6RXVPikS8wf0/edit#).
 Please contact the vendor to get up to date instructions for migrating from dockershim.
 
+## Migration from DockerShim
```

Comment:
One nit I spotted:
```suggestion
## Migration from dockershim
```

We usually write dockershim in all-lowercase.

Reply:
Good to know, renamed dockershim in lowercase everywhere.